examples/erc721_events_decoded.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+/*
+ This is a fun example of decoding ERC721 events with the built-in decoder.  We use the Clovers.network contracts as an example, which are tokens that are minted by players who work out a symetric othelo game. View more at https://clovers.network
+*/
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"time"
+
+	hypersyncgo "github.com/enviodev/hypersync-client-go"
+	"github.com/enviodev/hypersync-client-go/contracts"
+	"github.com/enviodev/hypersync-client-go/decoder"
+	"github.com/enviodev/hypersync-client-go/logger"
+	"github.com/enviodev/hypersync-client-go/options"
+	"github.com/enviodev/hypersync-client-go/types"
+	"github.com/enviodev/hypersync-client-go/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"go.uber.org/zap"
+)
+
+func main() {
```

Comment:
I wasn't sure what this error is about... The code runs fine at the very least. Didn't research it.

Reply:
This is related to golang compiler, without build tags you cannot have two main() functions in the same project. Main is used as main access endpoint to applications so it's natural that it fails. Should be or sub folder for this particular example or ignored via build tags + executed via specific build tag.